src/Devices/Ethernet_TC.h

```diff
@@ -0,0 +1,38 @@
+/*
+Based off Arduino's Ethernet library
```

Comment:
Use the following comment style. For at the start of files, classes and functions.

```
/**
 *
 */
```